### PR TITLE
add release notes step to release and make sure to include breaking changes

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: RELEASE.md [hunk 1]
```diff
@@ -3,6 +3,32 @@
...
+
+Habitat <VERSION> Release Notes
+
+We are happy to announce the release of Habitat <VERSION>. We have a number of new features as well as bug fixes, so please read on for all the details. If you just want the binaries, head on over to [Install Habitat](https://www.habitat.sh/docs/using-habitat/#install-habitat). Thanks again for using Habitat!
+
+Highlights: 
+* [BREAKING CHANGE] SOME BREAKING CHANGE
+```
+
+If you happen to be aware of features or bugs and feel there are user impacting words that need to be expressed. This is your opportunity to express them. Tasha will finalize and publish this PR after the release process is complete.
```
Comment (raskchanky) [inline RELEASE.md]: The first two sentences feel like one sentence that was accidentally split into two.

**Comment by raskchanky** [comment]
One small grammar comment but otherwise looks great!

File: RELEASE.md (hunk 1 above)
Comment (mwrock) [reply RELEASE.md]: hoo boy. I can't english.

**Comment by tashimi** [comment]
image

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/344525701).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/344525701).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**